Makefile

```diff
@@ -98,7 +98,8 @@ docs: manifests docs-dependencies crd-to-markdown ## Build docs
 	$(CRD_MARKDOWN) -f api/v1beta1/common.go -f api/v1beta1/openstackdataplaneservice_types.go -f api/v1beta1/openstackdataplanenodeset_types.go -f api/v1beta1/openstackdataplanedeployment_types.go -n OpenStackDataPlaneService -n OpenStackDataPlaneNodeSet -n OpenStackDataPlaneDeployment > docs/assemblies/custom_resources.md
 	bundle exec kramdoc --auto-ids docs/assemblies/custom_resources.md && rm docs/assemblies/custom_resources.md
 	sed -i "s/=== Custom/== Custom/g" docs/assemblies/custom_resources.adoc
-	cd docs; $(MAKE) html
+	cd docs; $(MAKE) html BUILD=upstream
```

Comment:
I would put this change into it's own PR if possible. It isn't directly related to the feature proposed.

Reply:
I need this on this PR so the CI can render the downstream version of the docs: https://logserver.rdoproject.org/23/723/6960badc997091a6fd3ac12d0a82d322477d8249/github-check/dataplane-operator-docs-preview/8be5764/docs_build/docs_build/dataplane/

Reply:
Ok. If there is no other way to address the issue.